### PR TITLE
Add blog post: Akka licence change one year later

### PR DESCRIPTION
**Pull request by njlbenn**

### CODE REVIEW

File: posts/2023-10-27-akka-licence-change-one-year-later.adoc [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+
+At the time, the announcement generated a lot of reaction, including the decision by a part of the open-source community to create a fork of Akka. The forked project, called Apache Pekko, https://www.mail-archive.com/announce@apache.org/msg08350.html[announced its first major release (v1.0.0) in July 2023].
+
+Recently at Lunatech, at the request of many of our clients, we have decided to revisit this topic, specifically with an objective to help our clients manage their exposure to the new BSL licence and, if necessary, to help them eliminate this exposure entirel. We have decided to share some of our thoughts on the topic in case they are valuable to a wider audience.
```
Comment (PeuTit) [inline posts/2023-10-27-akka-licence-change-one-year-later.adoc:11]: entirel -> entirely

File: posts/2023-10-27-akka-licence-change-one-year-later.adoc [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+
+Note that since the licence change there has been 1 critical back-port (https://github.com/akka/akka/releases/tag/v2.6.21[Release v2.6.21 · akka/akka]), which, incidentally has also been picked up by the Apache Pekko project (https://github.com/apache/incubator-pekko/issues/442[include critical TLS fix from Akka 2.6.21 (pekko-stream TLSActor) · Issue #442 · apache/incubator-pekko])
+
+Apache Pekko is proposed as a potential solution to mitigate the risk of either being on a version of Akka that will no longer receive critical security updates or of inadvertently depending on BSL code.
+
+It is worth noting, however, a possible migration to Pekko is not without its own risks (and I don’t just mean the general risks of migrating or rewriting software). By migrating to Pekko, one would be betting that the Apache Pekko project will be able to continue maintaining and evolving the framework - including security and bug fixes, and new functionality.
```
Comment (PeuTit) [inline posts/2023-10-27-akka-licence-change-one-year-later.adoc:73]: ... noting, however, **that** a possible ...

File: posts/2023-10-27-akka-licence-change-one-year-later.adoc [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+
+=== Paying for the licence is a valid option
+
+Paying for continued Akka use and future features is probably reasonable for complex applications that rely on the advanced distributed computing platform that Akka offers.
+
+In that case, the first thing you should do is speak to Lightbend to see how much it is actually going to cost you to stick with Akka and not have to make any changes at all. If it turns out to be more cost effective to come to a negotiated agreement with Lightbend for a commercial licence rather than paying the development costs to migrate away from Akka to an alternative solution, then paying for the licence is indeed a valid option. This is especially the case for those with complex distributed computing problems that rely on the proven features of Akka and would benefit from continued evolution of the framework.
```
Comment (PeuTit) [inline posts/2023-10-27-akka-licence-change-one-year-later.adoc:107]: cost effective -> cost-effective

File: posts/2023-10-27-akka-licence-change-one-year-later.adoc [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+
+=== Be clear about the tradeoffs if migrating
+
+For those organisations that have decided that migrating away from Akka is a better choice than paying for a commercial licence, it is worth asking "Is your Akka system being actively developed? Are you adding new features or is it in maintenance mode?"
+
+For applications that are in maintenance mode or where there are only the occasional evolutions and new features, migrating from Akka to Apache Pekko is likely to be the smoothest option. Of course there will still be a need to put resources into the initial migration, but consulting https://pekko.apache.org/docs/pekko/current/project/migration-guides.html[the Apache Pekko Migration Guide] we see that it is largely a 'Find/Replace' operation. However, the tradeoff for a relatively smooth initial migration might be that, given that Apache Pekko is a newly formed community, the pace of evolution and bringing new features might be slower than with Akka or other frameworks with more established communities and roadmaps.
```
Comment (PeuTit) [inline posts/2023-10-27-akka-licence-change-one-year-later.adoc:113]: Add a comma after 'Of course'

**Review by PeuTit — CHANGES_REQUESTED** [review]
Good write up, left a few minor comments, let me know if they are relevant or not!

File: posts/2023-10-27-akka-licence-change-one-year-later.adoc [hunk 5]
```diff
@@ -0,0 +1,130 @@
+= Akka Licence Change (One Year Later)
+njlbenn, eamelink, thinkmorestupidless
+v1.0, 2023-10-27
+:title: Akka Licence Change (One Year Later)
```
Comment (leo-bogastry) [inline posts/2023-10-27-akka-licence-change-one-year-later.adoc:4]: we usually also add the language tag here `:lang: en`

**Review by PeuTit — APPROVED** [review]

**Review by leo-bogastry — APPROVED** [review]

**Review by leo-bogastry — APPROVED** [review]

**Pull request merged**